README.md

```diff
@@ -60,16 +60,18 @@ I have tested Raven on Debian-Stretch and Ubuntu 16.04. Contributions to support
 
 ### Installing
 
+You will need at least Go 1.8 to build, and [Go's dep](https://github.com/golang/dep) package manager.  Before starting, make sure you have your $GOPATH set (if you do not we recommend: `echo "export GOPATH=~/go" >> ~/.bashrc && source ~/.bashrc`).
+
 ```shell
+mkdir -p $GOPATH/src/github.com/rcgoodfellow/
```

Comment:
so I think we can clean this up a bit by
```shell
go get github.com/rcgoodfellow/raven
cd $GOPATH/src/github.com/rcgoodfellow/raven/
```

Reply:
addressed in: b0e336a